torchvision/prototype/datasets/_builtin/__init__.py

```diff
@@ -5,10 +5,11 @@
 from .coco import Coco
 from .cub200 import CUB200
 from .dtd import DTD
+from .eurosat import EuroSAT
 from .fer2013 import FER2013
 from .gtsrb import GTSRB
 from .imagenet import ImageNet
-from .mnist import MNIST, FashionMNIST, KMNIST, EMNIST, QMNIST
+from .mnist import EMNIST, FashionMNIST, KMNIST, MNIST, QMNIST
```

Comment:
Could you revert that? It looks like you are using `usort>=1.*`. Since I saw this on another PR as well, could you explain how you installed it?

Reply:
I had some auto formatter enabled in vscode, that's how these changes were made. `save without formatting` seemed to work to get rid of these